### DISCUSSION TITLE
PKsim- how to decrease absorption

### DISCUSSION THREAD

**Discussion by jyna6** [community/community#138218] [Programming Help]
I build a PBPK model using PKsim.
The model is predicting too fast and too high absorption of the drug.
What factors should I modify to improve the model?

Thanks in advance.
Joo Young

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @jyna6 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by tpmst** [comment]
When you encounter issues with a PBPK (Physiologically Based Pharmacokinetic) model predicting too fast and too high absorption of a drug, several factors within the model might need adjustment. PBPK models are complex and highly detailed, integrating numerous physiological, biochemical, and drug-specific parameters. Here are some factors and steps you can consider modifying to refine your absorption predictions in PK-Sim:

1. Gastrointestinal (GI) Transit Times
2. Solubility and Dissolution Rate
3. Permeability
4. Particle Size and Formulation
5. Specific Absorption Sites
6. Physiological Parameters
7. Enzyme and Transporter Interactions
8. Model Validation with Experimental Data
9. Software-Specific Settings
10. Consult the Literature and Experts
